nsxt/resource_nsxt_policy_tier0_gateway.go

```diff
@@ -109,6 +109,17 @@ func resourceNsxtPolicyTier0Gateway() *schema.Resource {
 				Computed: true,
 				ForceNew: true, // Modification of transit subnet not allowed after Tier-0 deployment
 			},
+			"vrf_transit_subnets": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
```

Comment:
I'm guessing we expect the API to allow more subnets in future (such as ipv6 support?)
In this case, I would remove the `MaxItems` here to allow native support for this in the future, and rely on NSX validation in the time being.
If we don't expect the list to grow, we should remove the list here as well, to avoid confusion.

Reply:
sounds good